frontend/src/components/Table/Table.tsx

```diff
@@ -405,7 +406,7 @@ const Table = <T extends object>(props: PropsWithChildren<TableProps<T>>): React
       {!props.hideToolbar && (
         <div className="table-toolbar">
           {props.pageSize !== -1 && <TablePagination<T> instance={instance} />}
-          {!props.lockPageSize && props.data.length > 0 && (
+          {!props.lockPageSize && props.data.length > 0 && !props.hidePageSizeSelector && (
```

Comment:
is there a difference between lockPageSize and hidePageSelector?

Reply:
did not even notice we had that, changing now